plugins-base/XSFeatureTimezone.py

```diff
@@ -109,6 +110,7 @@ def HandleCityChoice(self,  inChoice):
         Layout.Inst().PopDialogue()
         try:
             data.TimezoneSet(city)
+            time.tzset()
```

Comment:
I think it would be nicer if this were part of `TimezoneSet` but I guess this is fine for now.